catalogue/webapp/components/WorkDetails/WorkDetails.tsx

```diff
@@ -253,30 +253,28 @@ const WorkDetails: FunctionComponent<Props> = ({ work, itemUrl }: Props) => {
     <>
       {onlineResourcesPrototype && onlineResources.length > 0 && (
         <WorkDetailsSection headingText="Available online">
-          <h3
```

Comment:
I know we have the `available online` heading, but should we be labelling the list for accessibility reasons?

Reply:
Maybe worth asking @alicerichmond about how we check that our lists are appropriately labelled in accessibility testing

Reply:
I think part of the move to removing 'Online resources' was that it didn't really add any useful information (potentially just added confusion). But that's not to say that there isn't a more appropriate heading we could find that would add some value.